components/help/help_controller.tsx

```diff
@@ -18,47 +17,46 @@ type Props = {
     };
 }
 
-export default class HelpController extends React.PureComponent<Props> {
-    public componentDidUpdate(): void {
-        const helpControllerNode = ReactDOM.findDOMNode(this);
+export default function HelpController(props: Props) {
+    const containerRef = useRef<HTMLDivElement>(null);
 
-        if (helpControllerNode && helpControllerNode instanceof HTMLDivElement) {
-            helpControllerNode.scrollIntoView();
-        }
-    }
+    useEffect(() => {
+        containerRef.current?.scrollIntoView();
```

Comment:
nit: We do have a package to support smooth scrolling and `scrollIntoViewIfNeeded` that is being used in the auto-height-switcher component. If this is something we want to implement here as well. :D

Reply:
We don't actually want smooth scrolling here. I forget the exact reason for this, but I believe it was something related to scrolling to the top when this page loads. Also, these pages are actually inaccessible in the app, and I'm going to file a ticket to remove them